packages/playwright-core/src/client/playwright.ts

```diff
@@ -73,4 +73,10 @@ export class Playwright extends ChannelOwner<channels.PlaywrightChannel> {
   static from(channel: channels.PlaywrightChannel): Playwright {
     return (channel as any)._object;
   }
+
+  async _startMockingProxy() {
+    const requestContext = await this.request._newContext(undefined, this._connection.localUtils()._channel);
```

Comment:
Forgot to pass the options?

Reply:
as discussed, this is on purpose and it respects only the default context options.